app.rb

```diff
@@ -32,8 +32,8 @@
   # Get basic info on all monitors.
   begin
     monitors = Pingometer.new(PINGOMETER_USER, PINGOMETER_PASS).monitors
-  rescue
-    @error_message = "Our status monitoring system, Pingometer, appears to be having problems."
+  rescue => error
+    @error_message = "Our status monitoring system, Pingometer, appears to be having problems. #{error}"
```

Comment:
I'm not sure if Pingometer will ever expose secrets in their error messages, but it was invaluable in debugging my credentials.